server/controllers/apiController.ts

```diff
@@ -122,6 +123,22 @@ const apiController: ApiControllerType = {
       return next(createErr('apiController', 'callCandidateMicroservice', err));
     }
   },
+
+  compareResults:(req,res,next) => {
+
+   
+   try{
+    //choose faster runtime
+    if(req.body.runtime < res.locals.candidateRuntime) res.locals.fasterRuntime = 'legacy';
```

Comment:
fasterRuntime doesn't appear to be used anywhere. Is this necessary?

Reply:
updated